list_service.rb

```diff
@@ -0,0 +1,46 @@
+def list_service(option)
+  case option
+  when 1
+    list_all_books
+    main
+  when 2
+    list_all_music_albums
+    main
+  when 3
+    list_all_games
```

Comment:
- [ ] Since you have called the `main` method in the select_option method as below:
```
def select_option(option)
  case option
  when 1..6
    list_service(option)
    main 
```
So I think you can delete it in the `list_service` and it works perfectly.

Reply:
That's true. Great catching that